### PR TITLE
dany's homework javacript-exercises

### PR DESCRIPTION
**Pull request by dany2kul**

### CODE REVIEW

File: week-1/E-strings/exercise.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var message="this is a string";
+var messageType=typeof message;
+console.log(messageType); 
+console.log(messageType);
```
Comment (panditita) [inline week-1/E-strings/exercise.js]: Your code is good but the expected result for this exercise is to log the message and the type, you're consol logging the type twice

Comment (panditita) [inline week-1/E-strings/exercise.js]: Your code is good but the expected result for this exercise is to log the message and the type, you're consol logging the type twice

File: week-1/F-strings-concatenation/exercise.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var greetingStart="hi,i'm called ";
+var name="daniella";
+var greeting=greetingStart + name;
+console.log(greeting);
```
Comment (panditita) [inline week-1/F-strings-concatenation/exercise.js]: It's a good practice to leave a space after the equal sign i.e. greetingStart = "hi, i'm called ";

Comment (dany2kul) [reply week-1/F-strings-concatenation/exercise.js]: notes taken.thanks

**Pull request closed without merging**